src/hooks/useConcentratorAum.ts

```diff
@@ -0,0 +1,48 @@
+import { Address } from "wagmi"
+
+import { VaultType } from "@/lib/types"
+import { useConcentratorId, useConcentratorTargetAssetId } from "@/hooks"
+import { useApiConcentratorDynamic } from "@/hooks/lib/api/useApiConcentratorDynamic"
+import useConcentratorAumFallback from "@/hooks/lib/tvl/concentrator/useConcentratorAumFallback"
+
+type ConcentratorAumProps = {
+  targetAsset: Address
+  primaryAsset: Address
+  ybToken: Address
+  type: VaultType
+}
+
+export function useConcentratorAum({
+  targetAsset,
+  primaryAsset,
+  ybToken,
+  type,
+}: ConcentratorAumProps) {
+  const { data: targetAssetId } = useConcentratorTargetAssetId({ targetAsset })
+  const { data: concentratorId } = useConcentratorId({
+    targetAsset,
+    primaryAsset,
+    type,
+  })
+
+  const apiQuery = useApiConcentratorDynamic({
+    targetAssetId,
+    concentratorId,
+    type,
+  })
+
+  const aumFallback = useConcentratorAumFallback({
```

Comment:
I think this is similar to the vaults where we just updated it to only read from chain for TVL. You can just do that here too IMO, no need for API.